src/main/com/aca/homework/week10/object/factory/ServletRequestObjectFactory.java

```diff
@@ -0,0 +1,10 @@
+package com.aca.homework.week10.object.factory;
+
+public class ServletRequestObjectFactory<ServletRequest> implements ObjectFactory<ServletRequest> {
```

Comment:
public class ServletRequestObjectFactory implements ObjectFactory<ServletRequest> will work too

Reply:
I added type parameter ``ServletRequest`` because I thought it would be a bad idea to use raw types for generic classes. Should I change it anyway?

Reply:
here you have defined a type parameter with the name ServletRequest. ServletRequest===T
you can just remove the first ServletRequest type parameter and write ServletRequest type next to ObjectFactory